src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,19 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { ConsoleLogger } from '../untils/consoleLogger.js';
+
+export const errorMiddleware = (error, req, res, next) => {
+  ConsoleLogger.error('Error: ', error);
```

Comment:
Consider including more details in the error log, such as the request URL or method, to aid in debugging.